llumnix/entrypoints/vllm/api_server.py

```diff
@@ -51,8 +53,8 @@
 
 async def _background_process_outputs():
     while True:
-        qsize = await request_output_queue.actor.qsize.remote()
-        request_outputs = await request_output_queue.actor.get_nowait_batch.remote(qsize)
+        qsize = request_output_queue.qsize()
```

Comment:
Can we avoid using a sync get method with sleep? Can we just make the get method async, then await it?